src/components/DevicesView.jsx

```diff
@@ -33,7 +43,16 @@ const DevicesView = ({ t, f, isFetching, devices }) => (
             </div>
             <div className={classNames(styles['set-content-cell'], styles['set-content-secondary'])} />
             <div className={classNames(styles['set-content-cell'], styles['set-content-secondary'])} />
-            <div className={classNames(styles['set-content-cell'])} />
+            <div className={classNames(styles['set-content-cell'])}>
+              <button
```

Comment:
The revoke button's style doesn't match the mockup -- it should be blue and aligned with the column title.
The style should still be there, the button was temporarily removed in [this commit](https://github.com/cozy/cozy-settings/commit/ee4d09cf7f9f3043a0ae1b17cf79ca4c8cdcc1e2).